features/addressing/managing_countries/editing_country_with_provinces.feature

```diff
@@ -0,0 +1,26 @@
+@addressing
+Feature: Editing country with provinces
+    In order to add or remove provinces in existing countries
+    As an Administrator
+    I want to be able to edit a country and its provinces
+
+    Background:
+        Given I am logged in as an administrator
+        And the store has country "United Kingdom"
+
+    @ui @javascript
+    Scenario: Adding province to existing country
+        Given I want to edit this country
+        When I add the "Scotland" province with "GB-SCT" code
+        And I save my changes
+        Then I should be notified about successful edition
+        And this country should have the "Scotland" province
+
+    @ui @javascript
+    Scenario: Removing province from existing country
+        Given this country has the "Nothern Ireland" province with "GB-NIR" code and "N.Ireland" abbreviation
+        And I want to edit this country
+        When I delete the "Nothern Ireland" province of this country
+        And I save my changes
+        Then I should be notified about successful edition
+        And this country should not have the "Nothern Ireland" province
```

Comment:
Can we have also a scenario for simultaneously deleting and adding provinces? I mean : a country has a province, I delete this province and I add another one. WDYT?